password_reset/forms.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.auth.forms import PasswordResetForm
+from django.core.exceptions import ValidationError
+from uplyft.models import CustomUser
+
+
+class EmailValidationOnForgotPassword(PasswordResetForm):
+    def clean_email(self):
+        email = self.cleaned_data["email"]
+        if not CustomUser.objects.filter(email__iexact=email, is_active=True).exists():
```

Comment:
Maybe we should have 2 checks here? 
if (email doesnt exist):
    error(No user registered with that email address)
else if email exists and not activated:
    error(You need to activate your account first, please check your email )

Something like that?